test/DebugInfo/DebugUnstructuredControlFlow.cl

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown -cl-std=CL2.0 -O0 -debug-info-kind=standalone -emit-llvm %s -o %t.ll
+// RUN: llvm-as %t.ll -o %t.bc
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_unstructured_loop_controls -spirv-text -o %t.spt
+// RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_unstructured_loop_controls -o %t.spv
+// RUN: llvm-spirv -r %t.spv -o %t.bc
+// RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+// Test that no debug info instruction is inserted between LoopControlINTEL and
+// Branch instructions. Otherwise, debug info interferes with SPIRVToLLVM
+// translation of structured flow control.
+//
+// Currently, Line DebugInfo instructions are still present between
+// LoopControlINTEL and Branch instructions. This does not affect SPIRVToLLVM
```

Comment:
Are you going to fix it later? What do you think about leaving FIXME/TODO here?

Reply:
Well, yeah, the line was confusing. Nothing to fix here.